bin/citgm-all.js

```diff
@@ -51,6 +64,15 @@ if (!lookup) {
   process.exit(1);
 }
 
+var cpus = os.cpus().length;
+if (app.autoParallel || (app.parallel && app.parallel > cpus)) {
+  app.parallel = cpus;
+  log.info('cores', 'running tests using ' + app.parallel + ' cores');
+}
+if (app.parallel && app.parallel > process.getMaxListeners()) {
```

Comment:
This needs to be `if (app.parallel && app.parallel + 1 > process.getMaxListeners()) {` as there always needs to be one more listener or you'll get the memory leak warnings